linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/org/apache/linkis/engineplugin/elasticsearch/conf/ElasticSearchConfiguration.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.linkis.engineplugin.elasticsearch.conf
+
+import org.apache.linkis.common.conf.{ByteType, CommonVars}
+
+object ElasticSearchConfiguration {
+
+  // es client
+  val ES_CLUSTER = CommonVars("wds.linkis.es.cluster", "127.0.0.1:9200")
+  val ES_DATASOURCE_NAME = CommonVars("wds.linkis.datasource", "default_datasource")
```

Comment:
The new configuration item name is recommended not to be prefixed with wds.

Reply:
thanks, fixed